dashboard/app/views/levels/editors/fields/_csa_fields.html.haml

```diff
@@ -3,5 +3,13 @@
 
 #csa_features.in.collapse
   .field
-    Select the desired CSA view:
-    = f.select :csa_view_mode, options_for_select([['Console', 'console'], ['Neighborhood', 'neighborhood'], ['Theater', 'theater']])
+    = f.label :csa_view_mode, 'CSA view mode'
+    = f.select :csa_view_mode, options_for_select([['Console', 'console'], ['Neighborhood', 'neighborhood'], ['Theater', 'theater']], @level.csa_view_mode)
+
+  .field
```

Comment:
If possible, have this display only when "Neighborhood" is selected. Or perhaps put it under a "Neighborhood" header.